specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/networkData.json

```diff
@@ -0,0 +1,361 @@
+              "readOnly": true,
+              "description": "The UTC start time of the actual traffic time range"
+            },
+            "endTime": {
+              "type": "string",
+              "format": "date-time",
+              "readOnly": true,
+              "description": "The UTC end time of the actual traffic time range"
+            },
+            "trafficDataState": {
+              "type": "string",
+              "description": "Indicate whether the Azure monitor agent is installed on the vm",
+              "enum": [
+                "Available",
+                "Not available"
+              ],
+              "x-ms-enum": {
+                "name": "trafficDataState",
+                "modelAsString": true,
+                "values": [
+                  {
+                    "value": "Available",
+                    "description": "The Azure monitor agent is installed on the vm"
+                  },
+                  {
+                    "value": "Not available",
+                    "description": "the Azure monitor agent is not installed on the vm"
+                  }
+                ]
+              }
+            },
+            "inboundConnectedResources": {
+              "$ref": "#/definitions/ActualTrafficConnectedResources"
+            },
+            "outboundConnectedResources": {
+              "$ref": "#/definitions/ActualTrafficConnectedResources"
+            }
+          }
+        },
+        "unusedPorts": {
+          "readOnly": true,
+          "type": "object",
+          "description": "The unused ports details",
+          "properties": {
+            "inboundConnectedResources": {
+              "$ref": "#/definitions/ConnectedResources"
+            },
+            "outboundConnectedResources": {
+              "$ref": "#/definitions/ConnectedResources"
+            }
+          }
+        }
+      }
+    },
+    "ConnectedResources": {
+      "readOnly": true,
+      "type": "object",
+      "description": "The connections details",
+      "properties": {
+        "tcpPortsCount": {
+          "readOnly": true,
+          "type": "integer",
+          "description": "The allowed connections tcp ports count"
+        },
+        "udpPortsCount": {
+          "readOnly": true,
+          "type": "integer",
+          "description": "The allowed connections udp ports count"
+        },
+        "tcpPorts": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The allowed connections tcp ports"
+        },
+        "udpPorts": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The allowed connections udp ports"
+        }
+      }
+    },
+    "ActualTrafficConnectedResources": {
+      "readOnly": true,
+      "type": "object",
+      "description": "The actual traffic details",
+      "properties": {
+        "trafficVolume": {
+          "readOnly": true,
+          "type": "string",
```

Comment:
how is the volume represented?  Does the user have to parse the string to understand it?

Reply:
Since volume can be large, it will be shown as "15 MB" or "1.2 GB" instead large number.

Reply:
I think this makes it much harder to consume for the user.  They have to parse a value instead of doing some simple math.  However, I don't consider it an API deal breaker.

Reply:
We actually had this as number and got feedback that human readable string is preferred.